### PR TITLE
[ErrorReport] Show full command in the error report

### PR DESCRIPTION
**Pull request by segiddins**

### CODE REVIEW

**Comment by orta** [comment]
I like this idea :+1:

**Comment by segiddins** [comment]
@kylef let me know if this looks good

File: lib/cocoapods/user_interface/error_report.rb [hunk 1]
```diff
@@ -137,6 +143,10 @@ def repo_information
             end
           end
         end
+
+        def original_command
+          "#{$0} #{ARGV.join(' ')}"
```
Comment (kylef) [inline lib/cocoapods/user_interface/error_report.rb]: Can we switch `$0` to `$PROGRAM_NAME` instead

**Comment by kylef** [comment]
Good to merge once feedback is addressed :+1:

**Pull request merged**